### DISCUSSION TITLE
Update matched scenes

### DISCUSSION THREAD

**Discussion by ghost** [stashapp/stash#2921] [General]
As scene metadata is constantly revised in stashdb, I'd like to update my scenes locally too.
(Corrections, missing performers added, better scene covers ...)

Is there any way to achieve this?

Of course data can pulled again in tagger view for scenes that are already matched, but that requires manual effort for scenes where all metadata is still identical.
Perhaps an additional filter would be nice, that hides all matched scenes with 100% identical metadata.
The identify task in stashs settings could potentially work too, if it could be applied on metadata filters (stashid not null) and not just based on folders.

Is there already a solution or being worked on?

**Comment by scruffynerf** [comment]
I've got this on my to-do list to write a plug-in for most of the above.

**Comment by MikeLund** [comment]
Some of the groundwork for this idea (https://github.com/stashapp/stash/issues/2903) is being done in https://github.com/stashapp/stash/issues/3553 for example.

In the meantime, you could try messing with the database for a quick one-off re-identifying of your scenes. Note that this obviously will remove all your existing stashids, so only do if you haven't manually identified many scenes/all your matched scenes were identified automatically. [Connect to the DB](https://github.com/stashapp/stash/wiki/Manually-Editing-the-Stash-Sqlite3-database) (after making a backup of course), and then run both these: `delete from scene_stash_ids; update scenes set organized = '0';`. Now the identify task should automatically identify all your scenes again; remember to choose "overwrite" options in the identify dialogue.
